frontend/static/js/app.js

```diff
@@ -207,10 +230,15 @@ Osgende.RouteList = function(map, container) {
                        autodividersSelector : div_func
     }).listview("refresh");
 
-    map.vector_layer.setSource(new ol.source.Vector({
-            url: Osgende.make_segment_url(data['results'], map.map),
-            format: new ol.format.GeoJSON()
-    }));
+    if (!Osgende.has_vector_tiles(map))
+    {
```

Comment:
Would you mind switching these if statements around everywhere, so that you get a positive condition, i.e.

if (Osgende.has_vector_tiles(map)) {
  // former else stuff
} else {
  // former if stuff
}

Also, bracket usually goes on the same line as if.